.github/workflows/ci.yml

```diff
@@ -477,9 +477,9 @@ jobs:
 
       - name: Java Samples
         run: |-
-          if [ $(find ./samples -name pom.xml | xargs grep -h "<kalix-sdk.version>" | tr -s [:space:] | uniq | wc -l) -ne 1 ]; then
+          if [ $(find ./samples -name pom.xml | xargs awk '/<version>/ {print; exit}') | tr -s [:space:] | uniq | wc -l) -ne 1 ]; then
             echo "Not all Java samples have the same SDK version:"
-            find ./samples -name pom.xml | xargs grep "<kalix-sdk.version>"
+            find ./samples -name pom.xml | xargs awk '/<version>/ {print; exit}')
```

Comment:
We don't kave `kalix-sdk.version` in the pom anymore. We need to match the first occurrence of `<version> instead.

And awk is better to match first occurrences.